definitions/pipelines/somatic_exome.cwl

```diff
@@ -5,6 +5,7 @@ class: Workflow
 label: "exome alignment and somatic variant detection"
 requirements:
     - class: SubworkflowFeatureRequirement
+    - class: StepInputExpressionRequirement
```

Comment:
Ideally we should be using the MultipleInputFeatureRequirement, but it was causing issues when I tried to test it on our bcbio workflow. It seems to be working for a small test case I put together to submit an issue to the Cromwell team, so I'm working on narrowing down the exact cause now. Fixing this would eliminate the need for the javascript expression in the top level workflow

Reply:
Ah, okay, I didn't make the connection that the issue you were mentioning was with the `MultipleInputFeatureRequirement`... thanks!